prowler/providers/github/services/repository/repository_service.py

```diff
@@ -117,6 +117,18 @@ def _list_repositories(self):
                                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
                             )
 
+                    secret_scanning_enabled = False
+                    try:
+                        if repo.security_and_analysis:
+                            secret_scanning_enabled = (
+                                repo.security_and_analysis.secret_scanning.status
+                                == "enabled"
+                            )
```

Comment:
What happens if `secret_scanning` is None?

Reply:
Good catch, I changed it to avoid `AttributeError` errors.